modules/3d-tiles/src/tile-3d-writer.ts

```diff
@@ -1,11 +1,11 @@
-import type {Writer} from '@loaders.gl/loader-utils';
+import type {Writer, WriterOptions} from '@loaders.gl/loader-utils';
 import {VERSION} from './lib/utils/version';
 import encode3DTile from './lib/encoders/encode-3d-tile';
 
 /**
  * Exporter for 3D Tiles
  */
-export const Tile3DWriter: Writer = {
+export const Tile3DWriter: Writer<unknown, never, WriterOptions> = {
```

Comment:
- The input data to this Writer should have a type that is also exported from the module.
- We should have a matching WriterOptions type even if it doesn't add anything.